board.py

```diff
@@ -39,3 +39,51 @@ def __setitem__(self, pos: str, item: Piece):
         row = int(pos[1]) - 1
         column = self.columns[pos[0]]
         self.matrix[row][column] = item
+
+    def inside_board(self, i: int, j: int):
+        return 0 <= i < 8 and 0 <= j < 8
+
+    def indeces_to_coordinates(self, i: int, j: int) -> str:
+        row = str(i + 1)
+        col = self.inv_col[j]
+        print(col + row)
+        return col + row
+
+    def coordinates_to_indeces(self, coord: str) -> tuple:
```

Comment:
```suggestion
    def coordinates_to_idxs(self, coord: str) -> Tuple[int, int]:
```
can do `from typing import Tuple` as well. All Tuples and Lists and Dicts and similar containers should have their types inside them as well. 

Sets, Dicts, Lists should all be imported from typing. The only things that shouldn't be imported are primitives (int, str, etc).

It is spelled `indices`, but it is common to abbreviate using `idx` and `idxs`

Reply:
You should also add docstrings for all of these functions. You won't remember the format of coordinate is if it is just a string

 https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html